src/dialects/postgres/query-generator-typescript.ts

```diff
@@ -75,4 +75,27 @@ export class PostgresQueryGeneratorTypeScript extends AbstractQueryGenerator {
       options?.cascade ? 'CASCADE' : '',
     ]);
   }
+
+  /**
+   * Generates an SQL query that returns all foreign keys of a table.
+   *
+   * @param   tableName The table or associated model.
+   * @returns           The generated SQL query.
+   */
+  // TODO: allow schema to be specified here as well
+  getForeignKeysQuery(tableName: TableNameOrModel) {
+    const table = this.extractTableDetails(tableName);
+    const quotedTableName = this.escape(table.tableName);
```

Comment:
about naming something "quoted", so far the sequelize codebase has used "quote" in an identifier context, and "escape" in a value context.

To follow that convention, this would be more aptly named `escapedTableName`